src/config/config.js

```diff
@@ -37,17 +37,17 @@ export const config = {
       position: 'Розробник ProZorro, засновник ГО «Електронна демократія», IT-євангеліст',
       topic: 'Чи можна електрифікувати демократію?',
       specialGuest: true,
-      link: '',
+      link: 'https://www.linkedin.com/in/flyonts',
       imageSrc: 'images/speakers/vf.jpg',
       bullets: []
     },
     {
       name: 'Дмитро Вербовий',
       position: 'UI engineer / Grid Dynamics',
-      topic: 'HTTP2',
+      topic: 'Key things that every front-end developer should know about HTTP/2. Why is it important to migrate?',
```

Comment:
better to omit "that"

"Key things every front-end developer should know about HTTP/2. Why is it important to migrate?"